config/karma.config.js

```diff
@@ -1,4 +1,4 @@
-var webpackConfig = require('./webpack.test');
+var webpackConfig = require('./webpack.test.js');
```

Comment:
I agree it should not need the .js, though it should still work with it. So you won't see any compile or runtime errors.